Assets/MixedRealityToolkit-SDK/Features/SpatialAwareness/System/Handlers/MixedRealitySpatialAwarenessSurfaceFindingHandler.cs

```diff
@@ -0,0 +1,5 @@
+﻿
+
```

Comment:
no namespace?

Reply:
odd.... not sure how that happened... i'll fix